scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<ul><li>Rephrase to "MOT challenge dataset, one of the most common benchmarking datasets for ..."</li><li>The goals might be a bit over-stated at the moment. Could just say "The goal of this notebook is to re-produce published results on the MOT challenge using the state-of-the-art FairMOT approach.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>